Components/Badge/badge.css

```diff
@@ -0,0 +1,187 @@
+.icon-badge-container
+{
+    width: 100%;
+    height: fit-content;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 3rem 1rem;
+}
+
+.icon-count-badge
+{
+    display: flex;
+    align-items:center;
+    position: relative;
+}
+
+.count-badge-5x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:  var(--notification-or-error);
+    border-radius: 100%;
+    color: white;
+    width: 2rem;
+    height: 2rem;
+    top: -25px;
+    right: -25px;
+    font-size: 1rem;
+}
+
+.count-badge-4x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:  var(--notification-or-error);
+    border-radius: 100%;
+    color: white;
+    width: 1.5rem;
+    height: 1.5rem;
+    top: -20px;
+    right: -20px;
+    font-size: 15px;
+}
+
+.count-badge-3x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:  var(--notification-or-error);
+    border-radius: 100%;
+    color: white;
+    width: 1.2rem;
+    height: 1.2rem;
+    top: -15px;
+    right: -15px;
+    font-size: 12px;
+}
+
+.count-badge-2x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:  var(--notification-or-error);
+    border-radius: 100%;
+    color: white;
+    width: 1rem;
+    height: 1rem;
+    top: -12px;
+    right: -12px;
+    font-size: 10px;
+}
+
+.status-badge-5x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
redundant lines of code... you can refactor like this 
```
.satus-badge-5x, count-badge-2x, ...{
    position: absolute;
    display: flex;
    justify-content: center;
    align-items: center;
}
```

Reply:
Done, Thanks for review. This advice was really helpful.